### DISCUSSION TITLE
Feature request: documentation for multisite server config on Nginx etc

### DISCUSSION THREAD

**Discussion by j-greig** [craftcms/docs#290] [Ideas]
I know it's impossible to cover every stack/scenario — but some basic documentation on configuring the most typical server setups for Craft multisite would be hugely appreciated.

(I've just spent more than an entire day getting multisite working via Nginx on a Forge-provisioned VPS and have [almost](https://craftcms.stackexchange.com/questions/32874/nginx-config-cachebusting-rewrite-for-multiple-server-blocks-domains) cracked it now, but am sure other people would welcome this too).

**Comment by angrybrad** [comment]
Yup, will leave this open, but this is on our list, just FYI.

**Comment by j-greig** [comment]
Thanks @angrybrad appreciate the update.
